packages/connect/src/SmoldotProvider/SmoldotProvider.test.ts

```diff
@@ -126,15 +134,13 @@ test('emits connect and never emits disconnect for development chain', async ()
 test('send formats JSON RPC request correctly', async () => {
   // we don't really care what the reponse is
   const responses =  ['{ "id": 1, "jsonrpc": "2.0", "result": "success" }'];
-  const rpcSend = sinon.spy();
+  const rpcSend = jest.fn();
```

Comment:
In order to use jest here you need to import it. There are 2 cases that `ReferenceError: jest is not defined` in this file

Reply:
Fixed